tests/unit/test_incremental.py

```diff
@@ -30,7 +30,8 @@ def setUp(self) -> None:
             'password': 'foo_pass',
             'port': 12345,
             'use_secondary': False,
-            'limit': None
+            'limit': None,
+            'offset': None
```

Comment:
is the offset testable with unit test?

Reply:
my understanding after checking the unit test code is that without basically rewriting the test suite it's not testable, and i don't really feel the energy to learn python unit testing and creating a whole new test suite for this feature 😆